change/@fluentui-react-tags-preview-7c233ae6-998d-44ce-811e-7f13798be096.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
would recommend a major bump here and go to 1.0.0, might as well use all of semver

Reply:
This is generated automatically from running `yarn nx workspace-generator prepare-initial-release  --project @fluentui/react-one-preview --phase=preview` - should this be command updated instead?

Reply:
oh that's interesting, let's keep as is then

Reply:
Actually personally I prefer minor because it's pre-release so having 0 as major version feels right